docs/example/assets/install-gcp.svg

```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm not sure why, but I can still see "OCH" in the SVG sources:
![Screenshot 2022-01-26 at 09 22 19](https://user-images.githubusercontent.com/7155799/151127773-eb9b9450-4629-4892-82b4-fda6ce72893d.png)

Please replace OCH occurrences for all 4 files, just for peace of mind 😄